python/ray/data/_internal/stats.py

```diff
@@ -262,6 +262,53 @@ def __init__(self, max_stats=1000):
             description="Seconds spent in iterator initialization code",
             tag_keys=iter_tag_keys,
         )
+        self.iter_total_s = Gauge(
+            "data_iter_total_seconds",
+            description="Total time spent in iteration",
+            tag_keys=iter_tag_keys,
+        )
```

Comment:
did you want to also add a chart for this, or just track the metric via the class and access it elsewhere? i didn't see one, i may have missed it